### ISSUE TITLE
Project layout and planning

### ISSUE THREAD

**Issue by floogulinc** [HexxiumCreations/threat-list-app#1]
We need to discuss the project's layout and plan how it will be created.

**Comment by floogulinc** [comment]
I think both projects should be able to get their block list from the same list that is used for those with adblock extensions.

For the app I think we can write it in Python and use [this parser](https://github.com/scrapinghub/adblockparser) to process incoming URLs and check them for matches. I was also looking at using parts of [this project](https://github.com/ParkerK/selfrestraint) to actually handle the blocking

**Comment by GilbertGobbels** [comment]
I agree, in that case we need an empty txt file with ONLY the domains and IP's that should be blocked.

**Comment by floogulinc** [comment]
Thats basicaly what [this](https://github.com/HexxiumCreations/threat-list/blob/gh-pages/hexxiumthreatlist.txt) is, except it uses the adblock list syntax, which is ok because there are already parsers for it.

**Comment by GilbertGobbels** [comment]
Coolio
